pymilvus/client/types.py

```diff
@@ -1100,3 +1100,18 @@ def to_dict(self) -> Dict[str, Any]:
         result = {"name": self.name}
         result.update(self.properties)
         return result
+
+
+class AnalyzeResult:
+    @property
+    def name(self) -> str:
+        return self._name
+
+    def __init__(self, info: milvus_types.AnalyzerResult) -> None:
+        self.tokens = info.tokens
+
+    def __str__(self) -> str:
+        return f"{{tokens: {self.tokens}}}"
```

Comment:
Is this double {{}} output as designed?
```
{{tokens: xxxx}}
```

Reply:
{{ means '{' in f"",
![img_v3_02jl_ef184b3a-3a37-46a5-a130-ebd2dd218a2g](https://github.com/user-attachments/assets/8bbfddad-8264-4fc7-b5aa-0edaa1dfdb92)